aspnetcore/mvc/controllers/bind-tryparse/7.0-samples/BindUsingTryParse/BindTryParseMVC/Models/DateRange.cs

```diff
@@ -26,8 +23,8 @@ public DateRange(DateOnly fromDate, DateOnly toDate)
             
             var range = value.Split('-');
 
-            if (!DateOnly.TryParse(range[0], provider, out var fromDate) 
-             || !DateOnly.TryParse(range[1], provider, out var toDate))
+            if (!DateOnly.TryParse(range[0],  out var fromDate) 
```

Comment:
removing the provider doesn't demonstrate the second tryparse method,
`public static bool TryParse(string value, IFormatProvider provider, T out result);`

Reply:
@fiyazbinhasan great catch. I removed it because when I added the experimental package, I was getting compile errors with it. Reverted!